pkg/storaged/clevis-dialogs.jsx

```diff
@@ -20,40 +20,84 @@
 import cockpit from "cockpit";
 import React from "react"; React;
 
-import { dialog_open, TextInput, PassInput } from "./dialogx.jsx";
+import { dialog_open, SelectOne, TextInput, PassInput, CheckBox } from "./dialogx.jsx";
 
 const _ = cockpit.gettext;
 
 export function add(client, block) {
     dialog_open({ Title: _("Add network key"),
                   Fields: [
-                      TextInput("url", _("Key server address"),
-                                { validate: val => !val.length && _("Server address cannot be empty") }),
+                      SelectOne("method", _("Method"), { },
+                                [ { value: "tang",
+                                    title: _("\"tang\" Binding server") },
+                                  { value: "http",
+                                    title: _("\"http\" Key escrow") }
+                                ]),
```

Comment:
Should this get some explanatory popup help? These are fairly technical slang, and not self-explanatory what the difference is.

Reply:
Yeah.  We need to seriously bikeshed the terminology.  I kinda made it awful on purpose for the playground.